FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -4,9 +4,20 @@
 
 import Foundation
 
+public struct FeedImageListRemote: Decodable {
+	public let items: [FeedImageRemote]
```

Comment:
Since `FeedImageListRemote` should be private, no need to make its property public, please remove `public` modifier
```suggestion
	let items: [FeedImageRemote]
```